examples/kubernetes/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
 }
 
 module "kubeflow" {
-  source = "WOGRA-AG/kubeflow/kustomization"
+  source = "../../"

```

Comment:
I know your problem with this line in the sense of 'local development / usage experience'. The  original thought of this example was to show a user how to use the published module. And yes, when I did changes I also changed this line as you did. Perhaps there should be a second folder, maybe 'local'?